pkg/tso/config.go

```diff
@@ -15,26 +15,37 @@
 package tso
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
 	"time"
 
+	"github.com/BurntSushi/toml"
 	"github.com/pingcap/log"
+	"github.com/pkg/errors"
 	"github.com/spf13/pflag"
 	"github.com/tikv/pd/pkg/utils/configutil"
+	"github.com/tikv/pd/pkg/utils/grpcutil"
 	"github.com/tikv/pd/pkg/utils/metricutil"
 	"github.com/tikv/pd/pkg/utils/typeutil"
 	"go.uber.org/zap"
 )
 
-const (
-	// defaultTSOUpdatePhysicalInterval is the default value of the config `TSOUpdatePhysicalInterval`.
-	defaultTSOUpdatePhysicalInterval = 50 * time.Millisecond
-)
-
 // Config is the configuration for the TSO.
 type Config struct {
 	BackendEndpoints string `toml:"backend-endpoints" json:"backend-endpoints"`
 	ListenAddr       string `toml:"listen-addr" json:"listen-addr"`
 
+	Name              string `toml:"name" json:"name"`
+	DataDir           string `toml:"data-dir" json:"data-dir"`
```

Comment:
The same question as the resource manager, what we are going to save in this dir?

Reply:
It seems that we save log files under data dir. It reuses the logic in pd/server. After sync, we can have a separate pr to refactor this thing across all service modes.